README.md

```diff
@@ -11,3 +11,13 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
+
+
+## Answers
+To prove that $O(log_2n)$ is the same as $O(log_5n)$ we must prove that $log_2n$ and $log_5n$ are the same asymptoticly. As the input sizes grow towards infinity, the leading term dominates the growth rate of the function which means the lower terms and the constant factors do not affect the growth rate. This means that at large values, if there is some constant $c$ where $log_5n=c \cdot log_2n$ then asymptoticaly $O(log_2n)$ is the same as $O(log_5n)$. The change of base formula for logarithms is: $log_ba=\frac{log_ca}{log_cb}$ or $log_cb=\frac{log_ca}{log_ba}$. By rearanging the previous equation we get $c=\frac{log_5n}{log_2n}=log_52$ which is a constant. As we have proven $log_5n$ and $log_2n$ differ by a constant factor, then asymptoticly $O(log_2n)$ and $O(log_5n)$ must be the same.
```

Comment:
But lower order terms and constant factors do affect the growth rate -- $n$ and $10n$ grow very differently.